src/frontend/utils/dato.ts

```diff
@@ -15,20 +15,61 @@ export interface IPeriode {
 
 export const dagensDato = startOfToday();
 
-interface FormatterProps {
+export const tidenesMorgen = new Date(1000, 1, 1);
+
+export const tidenesEnde = new Date(3000, 1, 1);
+
+interface FormatterDateProps {
     dato?: Date;
     datoformat: Datoformat;
     defaultString?: string;
 }
-export const formatterDate = ({ dato, datoformat, defaultString = '' }: FormatterProps): string => {
+export const formatterDate = ({
+    dato,
+    datoformat,
+    defaultString = '',
+}: FormatterDateProps): string => {
     return dato && isValid(dato) ? format(dato, datoformat) : defaultString;
 };
 
-export const formatterDateTilIsoString = (dato?: Date): string =>
+export const formatterDateTilIsoString = (dato?: Date): IsoDatoString =>
```

Comment:
```suggestion
export const formaterDateTilIsoString = (dato?: Date): IsoDatoString =>
```
<img width="692" alt="image" src="https://github.com/navikt/familie-ba-sak-frontend/assets/17828446/4524cad9-b03d-4e2a-a6f6-7290dd4fa395">

Reply:
Hehe ja good catch, endrer alle steder 😅

Reply:
Eventuelt kan den hete `dateTilIsoString`